src/scss/components/_fingerprints.scss

```
@@ -33,7 +33,7 @@
                 &.active {
                     pointer-events: all;
                     background-color: #152616;
-                    color: $succes-color;
+                    color: var(--succes-color);
```

Looks like the css variable has the correct spelling where it's declared (success).
```suggestion
                    color: var(--success-color);
```